forge/test/mlir/mnist/training/test_training.py

```diff
@@ -91,3 +93,39 @@ def test_mnist_training():
             break
 
     print(f"Test (total) loss: {test_loss}")
+
+@pytest.mark.parametrize("batch_size", [1, 2, 16])
```

Comment:
Let's add a bit bigger batches that are commonly used as well (e.g. 64)

Reply:
Thanks! Can you comment this details on the #355 issue, and notify dev to update this test and check higher batch sizes as well? :))